website/src/pages/index.astro

```diff
@@ -1,88 +1,134 @@
 ---
 import BaseLayout from '../layouts/base/BaseLayout.astro';
 import DataInfo from '../layouts/base/footer/DataInfo.astro';
+import { BrandName } from '../layouts/base/header/Brand';
 import { organismConfig } from '../types/Organism';
 import { dataOrigins } from '../types/dataOrigins';
 import { Page } from '../types/pages';
 import { ServerSide } from '../views/serverSideRouting';
+
+/* eslint-disable @typescript-eslint/naming-convention */
+const iconMapping: { [key: string]: string } = {
+    'Sequencing efforts': 'mdi--test-tube',
+    'Single variant': 'mdi--magnify',
+    'Compare side-by-side': 'mdi--compare-horizontal',
+    'Browse data': 'mdi--database-outline',
+};
+/* eslint-enable */
 ---
 
 <style>
+    a {
+        @apply text-slate-500 hover:underline;
+    }
+
+    .items a {
+        @apply text-black hover:underline hover:decoration-4;
+    }
     h1 {
-        @apply mb-4 text-center text-3xl;
+        @apply text-center font-light;
     }
 
-    .description a {
-        @apply underline;
+    @media (max-width: 768px) {
+        .largerLogo img {
+            display: none;
+        }
+
+        .largerLogo div {
+            @apply mx-auto;
+        }
+        .largerLogo h1 {
+            @apply text-center text-6xl;
+        }
+
+        .largerLogo h2 {
+            @apply text-center text-2xl;
+        }
+
+        .infoSection h1 {
+            @apply text-2xl;
+        }
     }
 </style>
 
 <BaseLayout title='GenSpectrum Dashboards'>
-    <h1 class='mt-8'>Explore up-to-date pathogen genomic data</h1>
-    <div class='description mx-auto mb-8 max-w-[1400px] px-8'>
-        GenSpectrum…
+    <div class='largerLogo my-10 flex items-center justify-center'>
+        <img src='/virus-outline-colorful.svg' alt='colorful-logo' width='200' height='200' />
+        <div class='ml-10'>
+            <h1 class='mb-4 text-7xl'>
+                <BrandName fontsize='font-medium' />
+            </h1>
+            <h2 class='text-end text-3xl font-light text-slate-600'>Inspect pathogen genomic data</h2>
+        </div>
+    </div>
 
-        <ul class='ml-6 mt-2 list-disc'>
-            <li>
-                …is a fully interactive platform aiming to help scientists and public health professionals query and
-                analyze pathogen sequences.
-            </li>
-            <li>…is open source, the code is available on <a href='https://github.com/GenSpectrum'>GitHub</a>.</li>
-            <li>
-                …uses data from the <a href='https://www.insdc.org/'
+    <div class='infoSection mx-auto my-12 flex max-w-[1200px] px-8'>
```

Comment:
```suggestion
    <div class='infoSection mx-auto my-12 sm:flex max-w-[1200px] px-8'>
```
To get
![grafik](https://github.com/user-attachments/assets/7ba684ac-4dbd-45a1-b2a4-11276c9449a7)
instead of
![grafik](https://github.com/user-attachments/assets/5d14c8a4-ea30-4319-95f9-d73c4d997d25)

(which is IMO easier to read on small screens)